resources/compass-runtime-agent/values.yaml

```diff
@@ -1,10 +1,13 @@
 global:
+  containerRegistry:
+    path: eu.gcr.io/kyma-project
   images:
-    containerRegistry:
-      path: eu.gcr.io/kyma-project
-    runtimeAgent:
+    runtime_agent:
+      name: "compass-runtime-agent"
       version: "PR-11547"
-    runtimeAgentTests:
+  testImages:
+    runtime_agent_tests:
```

Comment:
I'd love to see `compass-runtime-agent` name as it's the real name of the component instead of just `runtime-agent`. The rest is fine ;>

Reply:
names were changed